npm/src/language-service/worker-browser.ts

```diff
@@ -23,7 +23,13 @@ export function messageHandler(e: MessageEvent) {
       {
         log.setLogLevel(data.qscLogLevel);
         wasm.initSync(data.wasmModule);
-        const languageService = new QSharpLanguageService(wasm);
+        // we pass in dummy arguments for `readFile` and `listDir` because the browser worker doesn't support the project system right now
+        const languageService = new QSharpLanguageService(
+          wasm,
+          () => Promise.resolve(null),
+          () => Promise.resolve([]),
+          () => Promise.resolve(null),
```

Comment:
I wonder if we should have `getManifest` return `null` like you did, but have `readFile` and `listDir` fail instead. If we ever try to do file i/o from the website/playground, it's a bug, and failing fast would help us catch the error quicker.